site/_includes/macros/youtube-playlist.njk

```diff
@@ -0,0 +1,111 @@
+
+{% macro youTubePlaylist(id) %}
+
+
+{% for playlist in youtubePlaylist %}
+  {% if playlist.id === id %}
+
+  <div class="gap-top-400">
+    <div class="playlist hairline rounded-lg width-full">
+      <div class="playlist__details rounded-lg">
+        <div class="playlist__details-inner rounded-lg">
+          <div class="playlist__thumbnail">
+            <img
+              src="{{playlist.thumbnail}}"
+              height="158"
+              width="316"
+              alt="Thumbnail for {{playlist.title}}"
+              class="rounded-lg"
+            />
+            <div class="playlist__play-all">
+                <a
+                href="https://www.youtube.com/watch?v={{playlist.videos[0].id}}&list={{playlist.id}}"
```

Comment:
This looks fairly complex, can we prebuild the URL in data/youtubePlaylist.js instead?